h3-quinn/src/lib.rs

```diff
@@ -238,6 +240,30 @@ impl<S: Session> quic::RecvStream for RecvStream<S> {
     }
 }
 
+#[derive(Debug)]
+pub struct ReadError {
+    cause: quinn::ReadError,
+}
+
+impl Display for ReadError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        self.cause.fmt(f)
+    }
+}
+
+impl Error for ReadError {
+    fn source(&self) -> Option<&(dyn Error + 'static)> {
+        // TODO: implement std::error::Error for quinn::ReadError
```

Comment:
Do we need this ? Should I open a PR on quinn ?

Reply:
It's not required. It's only useful if users would want to be able to inspect for some specific variant or detail on the error.